index.html

```diff
@@ -1,657 +1,1213 @@
+                  <!-- SITE LOGO -->
+                  <div class="image site-logo">
+                    <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container/container_2037006891/image.coreimg{.width}.png/1669058748470/leaderimpactlogo.png" data-asset="/content/dam/LeaderImpact/site-./assets/leaderimpactlogo.png" data-title="logo" id="image-25b62a9953" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                      <a class="cmp-image__link" href="/content/leaderimpact/lang/en2.html" data-cmp-hook-image="link">
+                        <img src="./assets/logo.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="logo">
+                      </a>
+                    </div>
+                  </div>
+                  <!-- NAV MENU -->
+                  <div class="navigation nav-wrap">
+                    <nav id="navigation-241ab1aa9d" class="cmp-navigation" itemscope="" itemtype="http://schema.org/SiteNavigationElement">
+                      <ul class="cmp-navigation__group">
+                        <!-- MENU ITEMS -->
+                        <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                          <a href="/content/leaderimpact/lang/en2/join-a-group.html" class="cmp-navigation__item-link">Join a Group</a>
+                        </li>
+  
+                        <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                          <a href="/content/leaderimpact/lang/en2/start-a-group.html" class="cmp-navigation__item-link">Start a Group</a>
+                        </li>
+  
+                        <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                          <a href="/content/leaderimpact/lang/en2/resources.html" class="cmp-navigation__item-link">Resources</a>
+                        </li>
+  
+                        <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                          <a href="/content/leaderimpact/lang/en2/stories.html" class="cmp-navigation__item-link">About</a>
+                        </li>
+                      </ul>
                     </nav>
-                </div>
-                <div class="sub-nav">
-                    <span class="sub-nav-item">
-                        <a data-cmp-clickable="" class="nav-link-item" href="">Give</a>
-                    </span>
-                    <div class="nav-button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">Leaderhub</span>
+                  </div>
+                  <!-- SUB NAV -->
+                  <div class="container responsivegrid sub-nav">
+                    <div id="container-0b2a65237d" class="cmp-container">
+                      <!-- SUB NAV LINK -->
+                      <div class="link">

```

Comment:
@Prystal254 We can't use the link component. Please use a linked button instead.